src/AppDelegate.cpp

```diff
@@ -213,7 +213,7 @@ void AppDelegate::toggleDomainServer(bool start) {
 
 void AppDelegate::toggleAssignmentClientMonitor(bool start) {
     if (start) {
-        _acMonitorProcess->start(QStringList() << "--min" << "4");
+        _acMonitorProcess->start(QStringList() << "--min" << "5");
```

Comment:
Ahh ok thanks for the clarification, I thought this WAS the fix to the entity server problem. What was it then?